kogito-kafka-quickstart-springboot/README.md

```diff
@@ -71,8 +122,14 @@ To make use of this application it is as simple as putting a message on `travell
 
 this will then trigger the successful processing of the traveller and put another message on `processedtravellers` topic.
 
+* Send Message to Topic
+<p align="center"><img src="docs/images/sendMessagetoTravelersTopic.png"></p>
```

Comment:
better to spell out the command so that people can copy and paste, instead of making a picture